phpseclib/Crypt/RC2.php

```diff
@@ -339,7 +333,7 @@ public function setKey($key, $t1 = false)
         }
 
         if ($t1 < 1 || $t1 > 1024) {
-            throw new \LengthException('Key size of ' . $length . ' bits is not supported by this algorithm. Only keys between 1 and 1024 bits, inclusive, are supported');
+            //throw new \LengthException('Key size of ' . $t1 . ' bits is not supported by this algorithm. Only keys between 1 and 1024 bits, inclusive, are supported');
```

Comment:
Why was this changed?

Reply:
`$length` is an undefined variable.
As for why it was commented out, I'm not sure.

Uncommented.